### PR TITLE
add configuration option for datalayer hash encoding

### PR DESCRIPTION
**Pull request by SchiffFlieger**

### CODE REVIEW

File: w3cdatalayer/module.go [hunk 1]
```diff
@@ -17,3 +19,9 @@ func (m *Module) Configure(injector *dingo.Injector) {
 	flamingo.BindTemplateFunc(injector, "w3cDatalayerService", new(templatefunctions.W3cDatalayerService))
 	flamingo.BindEventSubscriber(injector).To(application.EventReceiver{})
 }
+
+func (m *Module) DefaultConfig() config.Map {
```
Comment (danielpoe) [inline w3cdatalayer/module.go]: Why not using the new CueConfig?

Comment (SchiffFlieger) [reply w3cdatalayer/module.go]: Yep, forgot about that. Updated.

File: w3cdatalayer/module.go [hunk 2]
```diff
@@ -17,3 +18,8 @@ func (m *Module) Configure(injector *dingo.Injector) {
 	flamingo.BindTemplateFunc(injector, "w3cDatalayerService", new(templatefunctions.W3cDatalayerService))
 	flamingo.BindEventSubscriber(injector).To(application.EventReceiver{})
 }
+
+// CueConfig schema and configuration
+func (m *Module) CueConfig() string {
+	return `w3cDatalayer: {	hashEncoding: string | *"base64url" }`
```
Comment (bastianccm) [inline w3cdatalayer/module.go]: prefix with `commerce: ` to follow the namespacing

Comment (SchiffFlieger) [reply w3cdatalayer/module.go]: Updated.

**Pull request merged**